libraries/botbuilder-schema/src/main/java/com/microsoft/bot/schema/models/MessageActivity.java

```diff
@@ -143,9 +143,9 @@ void setAttachments(List<Attachment> attachments) {
     /// <summary>
     /// Create an instance of the Activity class with IConversationUpdateActivity masking
     /// </summary>
-    public static ConversationUpdateActivity CreateConversationUpdateActivity()
+    public static MessageActivity CreateConversationUpdateActivity()
     {
-        return (ConversationUpdateActivity) new MessageActivity()
+        return (MessageActivity) new MessageActivity()
```

Comment:
consider writing this without the cast (which can throw) - instead use a local variable.